docs/reference/sql-tools.md

```diff
@@ -0,0 +1,289 @@
+            rs.getDouble("cpu_sys"));
+      }
+    }
+    ```
+
+    For the complete code of the demo, please refer to [here](https://github.com/GreptimeTeam/greptimedb-ingester-java/blob/main/ingester-example/src/main/java/io/greptime/QueryJDBC.java).
+  </TabItem>
+  <TabItem value="Go" label="Go">
+    The following code declares a GORM object model:
+    
+    ```go
+    type CpuMetric struct {
+        Host        string    `gorm:"column:host;primaryKey"`
+        Ts          time.Time `gorm:"column:ts;primaryKey"`
+        CpuUser     float64   `gorm:"column:cpu_user"`
+        CpuSys      float64   `gorm:"column:cpu_sys"`
+    }
+    ```
+    
+    If you are using the [High-level API](/user-guide/ingest-data/for-iot/grpc-sdks/go.md#high-level-api) to insert data, you can declare the model with both GORM and GreptimeDB tags.
+    
+    ```go
+    type CpuMetric struct {
+        Host        string    `gorm:"column:host;primaryKey" greptime:"tag;column:host;type:string"`
+        Ts          time.Time `gorm:"column:ts;primaryKey"   greptime:"timestamp;column:ts;type:timestamp;precision:millisecond"`
+        CpuUser     float64   `gorm:"column:cpu_user"        greptime:"field;column:cpu_user;type:float64"`
+        CpuSys      float64   `gorm:"column:cpu_sys"         greptime:"field;column:cpu_sys;type:float64"`
+    }
+    ```
+    
+    Declare the table name as follows:
+    
+    ```go
+    func (CpuMetric) TableName() string {
+      return "cpu_metric"
+    }
+    ```
+    
+    Use raw SQL to query data:
+    
+    ```go
+    var cpuMetric CpuMetric
+    db.Raw("SELECT * FROM cpu_metric LIMIT 10").Scan(&result)
+    ```
+  </TabItem>
+</Tabs>
+
+### Query library reference
+
+For more information about how to use the query library, please see the documentation of the corresponding library:
+
+<Tabs groupId="programming-langs">
+  <TabItem value="Java" label="Java">
+    - [JDBC Online Tutorials](https://docs.oracle.com/javase/tutorial/jdbc/basics/index.html)
+  </TabItem>
+  <TabItem value="Go" label="Go">
+    - [GORM](https://gorm.io/docs/index.html)
+  </TabItem>
+</Tabs>
+
+## Command line tools
+
+### MySQL
+
+You can use the `mysql` command line tool to connect to the GreptimeDB.
+Please refer to the [MySQL protocol](/user-guide/protocols/mysql.md) document for connection information.
+
+After you connect to the server, you can use all [GreptimeDB SQL commands](/reference/sql/overview.md) to interact with the database.
+
+### PostgreSQL
+
+You can use the `psql` command line tool to connect to the GreptimeDB.
+Please refer to the [PostgreSQL protocol](/user-guide/protocols/postgresql.md) document for connection information.
+
+After you connect to the server, you can use all [GreptimeDB SQL commands](/reference/sql/overview.md) to interact with the database.
+
+## GreptimeDB Dashboard
+
+You can run SQL and visualize data in the [GreptimeDB Dashboard](/getting-started/installation/greptimedb-dashboard.md).
+
+## GUI tools
```

Comment:
superset/metabase are not *sql tools*, we should use dbeaver here

Reply:
got it, I will remove the GUI tools section and add dbeaver in the future.